### ISSUE TITLE
Prop enableHardwareAcceleration isn't applied

### ISSUE THREAD

**Issue by xavierfoucrier** [Popmotion/stylefire#25]
Hi Stylefire team!

I am facing an issue with the `enableHardwareAcceleration` property: when set to false as specified [in the documentation](https://popmotion.io/stylefire/api/html/#html-styler-config-enablehardwareacceleration), the transform: translateZ(0) continue to be applied on the element, even if there is no CSS transform translateZ rules...

Here is the styler configuration:
```js
const character = Popmotion.styler(char, {
  enableHardwareAcceleration: false
});

Popmotion.tween({
  from: {
    y: '100%'
  },
  to: {
    y: 0
  },
  duration: 1000,
  ease: curves.text
}).start(character.set);
```

The element is a span that moves from '100%' to 0, Stylefire outputs:
```html
<span class="char" style="transform: translateY(0%) translateZ(0px);">A</span>
```

May be it's just a configuration problem...
Thanks for your help! :wink:

**Comment by InventingWithMonster** [comment]
Thanks for letting me know. This is published `stylefire@2.3.4`

**Comment by xavierfoucrier** [comment]
Hi @InventingWithMonster!

Thanks for the speed fix, but I am using stylefire with **popmotion "pure"**, not the "pose" version: it seem's that you have only updated the popmotion-pose package version instead of applying the fix to popmotion 8.5.4.

Best regards :beer:

**Comment by InventingWithMonster** [comment]
Ah good shout thanks, updated this in `popmotion@8.5.5`
